motiondetect.py

```diff
@@ -0,0 +1,75 @@
+import cv2 as cv
+from cv2 import imshow
+
+#works for images, recordings, live video 
+def rescaleFrame(frame, scale):
+    #scale the width and height, (cast to int)
+    width = int(frame.shape[1] * scale)
+    height =int(frame.shape[0] * scale)
+    dimensions = (width, height)
+    #return resize frame to particular dimension
+    return cv.resize(frame, dimensions, interpolation=cv.INTER_AREA)
+
+#convert frame to grey, compute Gaussian blur for noise reduction, update ave,
+#compute weighted averages, compute difference between wieghted ave and grayscale frame to get delta (background bodel - grayscale frame)
+def imgProcess(frame, avg):
+    #convert to grayscale image
+    gray = cv.cvtColor(frame, cv.COLOR_BGR2GRAY)
+    #blur image to reduce noise (filter size 21X21)
+    gray = cv.GaussianBlur(gray, (11,11),0)
```

Comment:
create variables to handle harcoded values so they are easier to change, especially for readabliity purposes